pkgs/development/libraries/pico-sdk/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation (finalAttrs: {
     hash = if (withSubmodules) then
       "sha256-GY5jjJzaENL3ftuU5KpEZAmEZgyFRtLwGVg3W1e/4Ho="
```

Comment:
This other `withSubmodules` hash is now

`sha256-fVSpBVmjeP5pwkSPhhSCfBaEr/FEtA82mQOe/cHFh0A=`

Reply:
Probably best to close this one and file your own PR.